exercise_perseus_render/assets/src/vue/perseus.vue

```diff
@@ -457,6 +464,9 @@
     .paragraph
       padding: 4px
 
+    .fixed-to-responsive
+      display: inline-block
+
```

Comment:
I agree that there is a large benefit to styling the exercises consistently with Kolibri. It makes the user experience more cohesive and removes an arbitrary boundary between content and application - much nicer.

However I don't think the true cost is just a "risk" of broken css. I'd argue that broken CSS is inevitable, sometimes in ways that we may never know because our users won't or can't communicate the problems to us...

The other big cost is long-term maintenance. If override private styles, we need to audit, test, and bug-fix every time we update Perseus, which makes updates more expensive. If we have to choose between staying current and staying stable, that's a major cost to us and our users.